cps/templates/feed.xml

```diff
@@ -49,26 +49,32 @@
       </author>
     {% endif %}
     {% if entry.publishers.__len__() > 0 %}
-      <publisher>
+      <dc:publisher>
         <name>{{entry.publishers[0].name}}</name>
-      </publisher>
+      </dc:publisher>
+    {% endif %}
+    {% if entry.pubdate %}
+      <dc:issued>{{entry.pubdate}}</dc:issued>
     {% endif %}
     {% for lang in entry.languages %}
-      <dcterms:language>{{lang.lang_code}}</dcterms:language>
+      <dc:language>{{lang.lang_code}}</dc:language>
     {% endfor %}
     {% for tag in entry.tags %}
     <category scheme="http://www.bisg.org/standards/bisac_subject/index.html"
               term="{{tag.name}}"
               label="{{tag.name}}"/>
     {% endfor %}
+    {% for identifier in entry.identifiers %}
+    <dc:identifier>{{identifier.val}}</dc:identifier>
+    {% endfor %}
     {% if entry.comments[0] %}<summary>{{entry.comments[0].text|striptags}}</summary>{% endif %}
     {% if entry.has_cover %}
     <link type="image/jpeg" href="{{url_for('opds.feed_get_cover', book_id=entry.id)}}" rel="http://opds-spec.org/image"/>
     <link type="image/jpeg" href="{{url_for('opds.feed_get_cover', book_id=entry.id)}}" rel="http://opds-spec.org/image/thumbnail"/>
     {% endif %}
     {% for format in entry.data %}
     <link rel="http://opds-spec.org/acquisition" href="{{ url_for('opds.opds_download_link', book_id=entry.id, book_format=format.format|lower)}}"
-          length="{{format.uncompressed_size}}" mtime="{{entry.atom_timestamp}}" type="{{format.format|lower|mimetype}}"/>
+          length="{{format.uncompressed_size}}" type="{{format.format|lower|mimetype}}"/>
```

Comment:
Calibre adds this entry to it's feed and many opds reader are more compatible to calibre than to the spec and some of them are very strict, if there is anything different form what they are expecting it fails. Last time I changed something several readers caused problems -> No change